src/lib/components/save-and-load/save-modal-btn.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	import { showModal, modalAction } from './store.svelte';
+</script>
+
+<button
```

Comment:
Would be nice to have this button slightly bigger on the sides and with a hover effect